packages/docusaurus-plugin-content-docs/src/lastUpdate.ts

```diff
@@ -32,9 +34,14 @@ export default function getFileLastUpdate(
   // Wrap in try/catch in case the shell commands fail (e.g. project doesn't use Git, etc).
   try {
     if (!shell.which('git')) {
-      console.log('Sorry, the docs plugin last update options require Git.');
+      if (!showedGitRequirementError) {
```

Comment:
how about 

warning it before the very first invoke of `lastUpdate` function ?

somewhere here https://github.com/facebook/docusaurus/blob/4fe6ae3c24f0df98887dbf9dee998e26ba78dd2e/packages/docusaurus-plugin-content-docs/src/metadata.ts#L106

Reply:
That will still make it be printed everytime someone updates the docs. And I don't want the `metadata.ts` to know about implementation details of `lastUpdate`.

Reply:
true. nvm this lgtm